### ISSUE TITLE
Unable to install MDX2JSON

### ISSUE THREAD

**Issue by nikitaeverywhere** [eduard93/Cache-MDX2JSON#5]
By executing
<code>do ##class(%Installer.Installer).InstallFromCommandLine("Z:\ZitRos\Work\MDX2JSON\Installer.cls.xml","Namespace=SAMPLES,SourceDir=Z:\ZitRos\Work\MDX2JSON", 0)</code>
in the terminal, I got this output:

Load started on 10/29/2014 13:15:46
Loading file Z:\ZitRos\Work\MDX2JSON\Installer.cls.xml as xml
Imported class: MDX2JSON.Installer
Compiling class MDX2JSON.Installer
Compiling routine : MDX2JSON.Installer.G1.MAC
Compiling routine MDX2JSON.Installer.1
Load finished successfully.
2014-10-29 13:15:46 0 MDX2JSON.Installer: Installation starting at 2014-10-29 13:15:46, LogLevel=0

Load of directory started on 10/29/2014 13:15:46

Loading file Z:\ZitRos\Work\MDX2JSON\Installer.cls.xml as xml
Imported class: MDX2JSON.Installer
Loading file Z:\ZitRos\Work\MDX2JSON\Utils.cls.xml as xml
Imported class: MDX2JSON.Utils
Loading file Z:\ZitRos\Work\MDX2JSON\REST.cls.xml as xml
Imported class: MDX2JSON.REST
Loading file Z:\ZitRos\Work\MDX2JSON\ResultSet.cls.xml as xml
Imported class: MDX2JSON.ResultSet

Compilation started on 10/29/2014 13:15:46 with qualifiers 'ck', compiling 4 classes, using 8 worker jobs
Compiling class MDX2JSON.Utils
Compiling class MDX2JSON.Installer
Compiling routine : MDX2JSON.Installer.G1.MAC
Compiling class MDX2JSON.REST
Compiling class MDX2JSON.ResultSet
Compiling routine MDX2JSON.REST.1
ERROR: Compiling class includes
ERROR:  MDX2JSON.REST.1(7) : MPP5635 : No include file 'MDX2JSON.MDX2JSON'
 TEXT: #include MDX2JSON.MDX2JSON
ERROR: Compiling method/s: GetDashboardsList
ERROR:  MDX2JSON.REST.1(8) : MPP5610 : Referenced macro not defined: 'ZENJSSTR'
 TEXT:          write $$$ZENJSSTR($System.Status.GetErrorText(ex.AsStatus()))
ERROR: Compiling method/s: GetWidgetsList
ERROR:  MDX2JSON.REST.1(3) : MPP5610 : Referenced macro not defined: 'get'
 TEXT:  set dashboard = $$$get("w")
ERROR: Compiling method/s: Http500
ERROR:  MDX2JSON.REST.1(8) : MPP5610 : Referenced macro not defined: 'ZENJSSTR'
 TEXT:     Write "{Error:"_$$$ZENJSSTR($System.Status.GetErrorText(pStatus))_"}" }
MDX2JSON.REST.1.INT(77) ERROR #1002: Invalid character in tag : '$$$ZENJSSTR($System.Status.GetErrorText(ex.AsStatus()))' : Offset:11 [zGetDashboardsList+5^MDX2JSON.REST.1]
 TEXT:          write $$$ZENJSSTR($System.Status.GetErrorText(ex.AsStatus()))
MDX2JSON.REST.1.INT(81) ERROR #1002: Invalid character in tag : '$$$get("w")' : Offset:20 [zGetWidgetsList+1^MDX2JSON.REST.1]
 TEXT:  set dashboard = $$$get("w")
MDX2JSON.REST.1.INT(93) ERROR #1002: Invalid character in tag : '"{Error:"_$$$ZENJSSTR($System.Status.GetErrorText(pStatus))_"}"' : Offset:23 [zHttp500+3^MDX2JSON.REST.1]
 TEXT:     Write "{Error:"_$$$ZENJSSTR($System.Status.GetErrorText(pStatus))_"}" }
MDX2JSON.REST.1.INT(94) ERROR #1044: PUBLIC label not allowed : 'public' : Offset:14 [zTest^MDX2JSON.REST.1]
 TEXT: zTest() public {
ERROR #5123: Unable to find entry point for method 'zTest' in routine 'MDX2JSON.REST.1'

> ERROR #5030: An error occurred while compiling class MDX2JSON.REST
> Compiling routine MDX2JSON.Installer.1
> Compiling routine MDX2JSON.ResultSet.1
> ERROR: Compiling class includes
> ERROR:  MDX2JSON.ResultSet.1(9) : MPP5635 : No include file 'MDX2JSON.MDX2JSON'
>  TEXT: #include MDX2JSON.MDX2JSON
> Compiling routine MDX2JSON.Utils.1
> ERROR: Compiling class includes
> ERROR:  MDX2JSON.Utils.1(7) : MPP5635 : No include file 'MDX2JSON.MDX2JSON'
>  TEXT: #include MDX2JSON.MDX2JSON
> Detected 11 errors during compilation in 0.082s.

ERROR #5475: Error compiling routine: MDX2JSON.REST.  Errors:
ERROR:  MDX2JSON.REST.1(7) : MPP5635 : No include file 'MDX2JSON.MDX2JSON'
 TEXT: #include MDX2JSON.MDX2JSON
Detected 11 errors during load.
2014-10-29 13:15:46 0 MDX2JSON.Installer: ERROR #5475: Error compiling routine: MDX2JSON.REST.  Errors:
ERROR:  MDX2JSON.REST.1(7) : MPP5635 : No include file 'MDX2JSON.MDX2JSON'
 TEXT: #include MDX2JSON.MDX2JSON
2014-10-29 13:15:46 0 MDX2JSON.Installer: Installation failed at 2014-10-29 13:15:46
2014-10-29 13:15:46 0 %Installer: Elapsed time .11046s

In short, compiler says that there's no include file 'MDX2JSON.MDX2JSON'.

How can I install this properly?
